packages/augur-ui/src/modules/app/components/app.tsx

```diff
@@ -534,18 +539,23 @@ export default class AppView extends Component<AppProps, AppState> {
           })}
         >
           <section className={Styles.App__loadingIndicator} />
-          <section
+
+          <section className={Styles.Main}>
+            <section
+              className={classNames(Styles.TopBar, Styles.TopBar__floatAbove)}
+              onClick={this.mainSectionClickHandler}
+              role="presentation"
+            >
+              <TopBar />
+            </section>
+
+            <section
             className={Styles.SideBar}
             onClick={e => this.mainSectionClickHandler(e, false)}
             role="presentation"
           >
-            <div className={Styles.Logo}>
-              <Link to={makePath(DEFAULT_VIEW)}>
-                <Logo />
-              </Link>
-            </div>
             {this.renderMobileMenuButton(unseenCount)}
-            <SideNav
+            {isMobile && <SideNav
```

Comment:
we are getting away from using `isMobile` is there a different way?

Reply:
Yup - fixing